src/NJsonSchema.CodeGeneration.TypeScript/NJsonSchema.CodeGeneration.TypeScript.csproj

```diff
@@ -5,7 +5,9 @@
     <Version>10.0.22</Version>
     <PackageTags>json schema validation generator .net</PackageTags>
     <Copyright>Copyright © Rico Suter, 2018</Copyright>
+    <!--
```

Comment:
Also use license type?

Reply:
I don't know what the tag is inside `csproj` files, only `nuspec` files.